### PR TITLE
tests: stable run_node

### PR DESCRIPTION
**Pull request by Hijiao**
Signed-off-by: Max <xiahaijiao@hotmail.com>

## What have you changed? (mandatory)

Keep binding  port in 2 MSL(Maximum Segment Lifetime).

## How has this PR been tested? (mandatory)
Unit test

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)
No
## Does this PR affect tidb-ansible update? (mandatory)
No

### CODE REVIEW

File: components/test_raftstore/src/server.rs [hunk 1]
```diff
@@ -169,7 +169,8 @@ impl Simulator for ServerCluster {
         });
         let cop = coprocessor::Endpoint::new(&server_cfg, store.get_engine(), cop_read_pool);
         let mut server = None;
-        for _ in 0..100 {
+        // try to bind port until 2 MSL (2 * 120 sec)
```
[the remark this answers is not in the document]
Comment (Hijiao) [reply components/test_raftstore/src/server.rs]: if not, CI test will fail randomly. Which one do you prefer? A longer test or a random error?😂

File: components/test_raftstore/src/server.rs [hunk 2]
```diff
@@ -169,7 +169,8 @@ impl Simulator for ServerCluster {
...
+        for _ in 0..240 {
```
Comment (overvenus) [inline components/test_raftstore/src/server.rs]: `0..50` is enough.

File: components/test_raftstore/src/server.rs [hunk 3]
```diff
@@ -186,7 +187,7 @@ impl Simulator for ServerCluster {
                 Some(Err(Error::Grpc(GrpcError::BindFail(ref addr, ref port)))) => {
                     // Servers may meet the error, when we restart them.
                     debug!("fail to create a server: bind fail {:?}", (addr, port));
-                    thread::sleep(Duration::from_millis(100));
+                    thread::sleep(Duration::from_millis(1000));
```
Comment (overvenus) [inline components/test_raftstore/src/server.rs]: 100ms is enough.

**Comment by breeswish** [comment]
Hmm. Why 2 MSL (not other value?)

**Comment by Hijiao** [comment]
@breeswish  TCP connection will be in a `TIME_WITE` state for at most 2ML in some cases.

**Pull request closed without merging**